src/test/unit_tests/test_nc_nsfs_health.js

```diff
@@ -41,6 +46,96 @@ mocha.describe('nsfs nc health', function() {
         await nb_native().fs.rmdir(DEFAULT_FS_CONFIG, bucket_storage_path);
     });
 
+    mocha.describe('nsfs nc health cli validations - should fail', function() {
```

Comment:
You can add 3 tests for argument with boolean ('false' and the uppercase of the 2 options).

Reply:
I think that all optional true/false values should be test-covered once since the validation functions are being re-used in health/manage CLI.
When looking at the tests I see that the tests checking for validation are scattered in the existing jest CLI account test / mocha CLI test and now in health tests.
I think we should have a single unit test file for the CLI arguments/types etc validations.
WDYT?

Reply:
I agree with you.

Reply:
@shirady I'll open an issue regarding tests refactoring, if you have more ideas I would love you to put it there as well :)